### PR TITLE
Restructure r4utils directory

### PR DESCRIPTION
**Pull request by AnjanaSenanayake**
Restructure r4utils directory to have separate child directories for util packages

### CODE REVIEW

File: utils/fhirpath/Package.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# FHIR R4 Utils FHIRPath package
+
+## Package Overview
+
+Package containing FHIR related processors and utilities for implementing FHIR Path related integrations.
+
+|                      |                      |
+|----------------------|----------------------|
+| FHIR version         | R4                   |
+| Implementation Guide | http://hl7.org/fhir/ |
```
Comment (sameeragunarathne) [inline utils/fhirpath/Package.md:10]: Shouldn't this be https://hl7.org/fhir/fhirpath.html?

File: utils/fhirpath/Ballerina.toml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[package]
+org = "ballerinax"
+name = "health.fhir.r4utils.fhirpath"
+version = "1.0.3-snapshot"
+distribution = "2201.5.0"
```
Comment (sameeragunarathne) [inline utils/fhirpath/Ballerina.toml:5]: let's update to 2201.6.0

File: utils/fhirpath/Ballerina.toml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+[package]
+org = "ballerinax"
+name = "health.fhir.r4utils.fhirpath"
+version = "1.0.3-snapshot"
```
Comment (sameeragunarathne) [inline utils/fhirpath/Ballerina.toml:4]: Let's not use snapshots

File: utils/fhirpath/Package.md (hunk 1 above)
Comment (AnjanaSenanayake) [reply utils/fhirpath/Package.md]: Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/08e57e7fae8edd666b262458106711c51a4fd471

File: utils/fhirpath/Ballerina.toml (hunk 2 above)
Comment (AnjanaSenanayake) [reply utils/fhirpath/Ballerina.toml]: Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/08e57e7fae8edd666b262458106711c51a4fd471

File: utils/fhirpath/Ballerina.toml (hunk 3 above)
Comment (AnjanaSenanayake) [reply utils/fhirpath/Ballerina.toml]: Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/08e57e7fae8edd666b262458106711c51a4fd471

**Review by sameeragunarathne — APPROVED** [review]

**Pull request merged**